studio/src/store.js

```diff
@@ -1,9 +1,9 @@
 import { Fragment } from './aem/fragment.js';
-import { getEditorPanel } from './editor-panel.js';
```

Comment:
editor-panel already imports the store api.
transitive updates causing problems.